Cargo.toml

```diff
@@ -14,15 +14,15 @@ http = "0.2"
 hyper = { version = "0.14", default-features = false, features = ["client"] }
 log = { version = "0.4.4", optional = true }
 rustls-native-certs = { version = "0.6", optional = true }
-rustls = { version = "0.20.1", default-features = false }
+rustls = { version = "0.20.7", default-features = false }
```

Comment:
This commit doesn't help, versions in `Cargo.toml` allow the use of semantically compatible new versions by default (and in fact, Cargo will prefer newer versions), so this change is meaningless here.

Reply:
It's explained here: https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-cratesio.